bob.js

```diff
@@ -1098,41 +1108,42 @@ username = (message) => {
 
 // Gets a formatted string for @ing a user
 mention = (user) => {
-    return "<@" + user.id + ">"; 
+    return "<@" + user.id + ">";
 }
 
 // Formats a time in seconds in H:mm:ss
 formatTime = (time) => {
     var hrs = Math.floor(time / 3600);
     var min = Math.floor((time - (hrs * 3600)) / 60);
-    var sec = time - (hrs * 3600) - (min * 60);
-    sec = Math.round(sec * 100) / 100;
+    var sec = Math.round((time - (hrs * 3600) - (min * 60)) * 100) / 100;
 
-    var result = (hrs < 10 ? "0" + hrs : hrs);
+    var result = hrs.toString();
     result += ":" + (min < 10 ? "0" + min : min);
     result += ":" + (sec < 10 ? "0" + sec : sec);
-    return result;
+    if (sec % 1 == 0) {
+        result += ".00";
+    } else if ((sec * 10) % 1 == 0) {
+        result += "0";
+    }
+    return result.replace(new RegExp("^[0:]*", ''), "");
```

Comment:
Correct me if I'm wrong, this just removes the hour portion if the time is <1 hour? This might make times in `!s` look weird next to each other if someone gets a sub 1 time and another gets over 1 hour, might be best to just keep it consistent.

Reply:
It doesn't only remove the hour portion, it removes everything up until the first non-zero digit. I can change it back for !s, but I think it would look good for the !me table